### DISCUSSION TITLE
Adding a license file on MacOS

### DISCUSSION THREAD

**Discussion by kovar-ursa** [Vector35/binaryninja-api#5531] [Q&A] [answered]
I am installing the latest version of Binary Ninja on my new Mac Air. I have a license file but a new install of BN isn't prompting me for it. Also, ~/Library/Application Support/Binary Ninja hasn't been created so I cannot manually drop it there. I'm unable to find an option in the UI to load the license file. The documentation all says that BN will prompt me for it.

Assistance would be quite welcome. Thanks!

**Comment by negasora** [comment, marked as answer]
It sounds like you may be running the free version, which doesn't require a license. Make sure to use the installer from the links in your purchase email.

**Reply by kovar-ursa** [reply]
Spot on. I realized this when I was out driving. Silly error, sorry.

**Reply by negasora** [reply]
No worries, glad it's an easy fix!
